Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1520,7 +1520,9 @@ public InvalidBlockException ValidateNextBlock(Block<T> block)
                     $"the block #{index - 1}'s ({prevTimestamp}).");
             }
 
-            if (block.ProtocolVersion > BlockMetadata.PoWProtocolVersion)
+            // Since the version cannot be downgraded,
+            // the target block is PBFT based block if the last block is PBFT based block.
+            if (lastBlock?.ProtocolVersion > BlockMetadata.PoWProtocolVersion)
```

Comment:
I think this logic needs to be checked again. `Block<T>.LastCommit` **must** be `null` on cases where we don't check for the validity of `LastCommit`.

Reply:
I tried to make policy to determine it, but there was a problem making totally new field to policy and make it pass. You can find this argument on discord https://discord.com/channels/928926944937013338/998498344416579636/1032935816944701540

Reply:
My bad. I forgot about that. Still looks a bit icky at the moment. 😥
As long as we fix this later down the line, I guess it's fine.